### ISSUE TITLE
Bug : Missing images after update to v20.0.0 (Cocoapods install)

### ISSUE THREAD

**Issue by the0neyouseek** [stripe/stripe-ios#1655]
## Summary

After updating the sdk to the latest version on my project I found that there are no longer any images displayed.

| Main View `STPPaymentOptionsViewController` | Adding a new Payment Option | 
|:-:|:-:|
| ![Screenshot 1](https://user-images.githubusercontent.com/4009175/94240355-79bb4900-ff13-11ea-8cc0-b040f4f285a7.PNG) | ![Screenshot 2](https://user-images.githubusercontent.com/4009175/94240398-8770ce80-ff13-11ea-887b-a1bf84687b4a.PNG) |

## Code to reproduce

Here's the code used to display the view controller :
```swift
let customerContext = STPCustomerContext.init(keyProvider: StripeKeyProvider.shared)
let paymentMethodsViewController = STPPaymentOptionsViewController(
	configuration: STPPaymentConfiguration.shared(),
	theme: .custom,
	customerContext: customerContext,
	delegate: self
)
self.navigationController?.pushViewController(paymentMethodsViewController, animated: true)
```

## iOS version

Tested on iOS 13-14

## Installation method

Cocoapods
```rb
pod 'Stripe'
```

## SDK version

Latest (`v20.0.0`)

**Comment by davidme-stripe** [comment]
Hey @the0neyouseek! I haven't seen this before, but we did move to Asset Catalogs in Stripe 20.0.0, so it's possible that we missed something.

To help debug, could you provide some additional info? Sorry for the trouble!
* Which version of Cocoapods are you using? `pod --version`
* Did you run `Product -> Clean Build Folder` in Xcode?
* In the `Pods` project, is the `Stripe-Stripe` bundle target being built, and does the `Build Phases -> Copy Bundle Resources` step contain `Stripe.xcassets`?
* In your compiled .app, does `Frameworks/Stripe.framework/Stripe.bundle` exist? Does a `Stripe.bundle` appear anywhere in the .app bundle?

**Comment by the0neyouseek** [comment]
Hi @davidme-stripe!
Thanks for the tips 😉 I deleted my pods folder, did a clean install and cleaned the build folder and it did the trick !

**Comment by davidme-stripe** [comment]
Thanks for the follow-up! I tested it on a real device (without moving the Stripe.xcassets from the Stripe-Stripe bundle to the Stripe target), and I'm seeing images. I wonder if something else was cached that only impacted device builds. I'll try a few other scenarios of upgrading from 19.x to reproduce this.

**Comment by the0neyouseek** [comment]
I'm on Xcode 12 targeting for iOS 12 min. if that helps

**Comment by davidme-stripe** [comment]
Hi! I think we've fixed this in version [21.2.0](https://github.com/stripe/stripe-ios/releases/tag/21.2.0), please give it a try.
